testing/test_testcase1.py

```diff
@@ -10,6 +10,7 @@
 import os
 import utilities
 import requests
+#from examples.python import twoday_p
```

Comment:
A reason this is commented and then repeated one line below?

Reply:
sorry for not restoring it. This repo assumes that we put the home direcotry to the PYTHONPATH, otherwise the correpsonding package can't be imported. Therefore I commented to facilitate the test.